### ISSUE TITLE
Jellyfin Server Software coming to NAS package centers (like Synology's)

### ISSUE THREAD

**Issue by essasetic** [jellyfin/jellyfin#293]
It would make it easier for the general user to get Jellyfin if it was in Synology's Package Centre (for example). It doesn't need to be added now but it would be something useful for the future.

**Comment by greee-fr** [comment]
Oh yes please, provide us a spk !!!

**Comment by guim31** [comment]
As I can't achieve the Docker installation, I hope a SPK is coming soon ^^

**Comment by anthonylavado** [comment]
The Hardware Acceleration is still a bit of a work in progress no matter what platform you're using, but the request has been noted!

Thanks for using Jellyfin :smile:

**Comment by travelar** [comment]
Would love to see a Synology build. The docker container runs very poorly on my Synology.

**Comment by poulpor** [comment]
I'm very interested in a Synology package in the future, using  docker container for the moment but I can't get steady hardware acceleration, it seems to start for some minutes then hangs (can see a -vaapi process with htop command). I also have extremely poor network performance in default bridge network mode, but a lot better in host mode.

**Comment by haan2787** [comment]
i like to see a synology package the docker version have some  things i do not like

**Comment by koelner21** [comment]
Hello,
Synology Support... please :)

**Comment by Aisub** [comment]
A Synology package would be amazing! I would love to test it out one day on an ds414.

**Comment by anthonylavado** [comment]
After 10.3.2 lands, I can start looking into this. I don't know that any of the core or support teams have any Synology hardware, so we would need someone to test.

I have applied here for more details: https://www.synology.com/en-global/support/developer

I am also looking at https://synocommunity.com/

**Comment by koelner21** [comment]
My englisch isn‘t very well. I can testing the Programm in my DS418play

**Comment by Aisub** [comment]
I did not expect such a fast response thank you very much. Maybe you could test it for yourself on an Xpenology system? (https://xpenology.org/). 

The synocommunity is a great way to spread the program to other Synology owners. I can only recommend it.

**Comment by koelner21** [comment]
One﻿ last thing:
:!:DO NOT UPDA﻿TE DSM BEYOND VERSION 6.1.7 with loader v1.02b.
IN OTHER WORDS  DO NOT UPDATE TO DSM 6.2:!:
You have been warned.﻿﻿

My Version is 6.2.1. Sorry.

**Comment by Essasetic** [comment]
> After 10.3.2 lands, I can start looking into this. I don't know that any of the core or support teams have any Synology hardware, so we would need someone to test.
> 
> I have applied here for more details: https://www.synology.com/en-global/support/developer
> 
> I am also looking at https://synocommunity.com/
I'll be willing to test. I have a DS218+ and I used VideoStation primarily for media (for now) anyway.

**Comment by WodKa78** [comment]
> After 10.3.2 lands, I can start looking into this. I don't know that any of the core or support teams have any Synology hardware, so we would need someone to test.
> 
> I have applied here for more details: https://www.synology.com/en-global/support/developer
> 
> I am also looking at https://synocommunity.com/

I am also available for testing. DS418J ( rtd1296 arch)

**Comment by Evpok** [comment]
Available to test on DS216j

**Comment by greee-fr** [comment]
OK for testing on an old DS713+ (with a DX513)

**Comment by poulpor** [comment]
will be happy to test it too on my xpenology DS918+ if it can be installed together with emby, which I suppose is not a good idea, but I'm stil using it for its reliable hardware transcoding.
With Jellyfin 10.3.1 installed on synology with docker,  vaapi transcoding is not consistent, even if it seems to work better than previous versions, especially with HEVC.

**Comment by koelner21** [comment]
I do not want to use the program for transcoding, since I have a 4K TV and therefore do not need it. My main concern is the skins and other addons.

**Comment by kozi48** [comment]
ok for testing on a xpenology 3615ds

**Comment by kozi48** [comment]
i can also test on DS218+

**Comment by Nedlinin** [comment]
Willing to test on a DS918+

**Comment by James-Jie** [comment]
i can also test on DS3615xs.

**Comment by moritzleick** [comment]
if/when i get my new DS918+ running, i would also be available for testing

**Comment by phoenix83** [comment]
Can test DS415+

**Comment by anthonylavado** [comment]
Thanks all, we need to work on some more packaging enhancements now that we're stable (kind of) on 10.3.2. I'll advise when we have something to go on.

**Comment by MartinH0** [comment]
Can you keep us updated when there will be a SPK File for Synology?
Also: do you still need testers? Have a DS1817+ (DSM 6.2.2)

What should I install?

**Comment by waylonwang** [comment]
I can test on DS916+ and DS216play (DSM 6.2.2)，thank you!

**Comment by intothevoid** [comment]
I have a DS918+ happy to help with testing builds. Cheers!

**Comment by Link2000** [comment]
My DS214play is ready for tests

**Comment by awp0** [comment]
Just want to comment for Syno DS918+ users (J3455 CPU with Quick Sync), that I seem to have HW Accelerated transcoding working within the standard Jellyfin docker image (version 10.3.7.0) on the DS918+. I was able to accomplish this by running it in privileged mode (not everyone will be comfortable with this), which exposes the host's /dev directory, including the required renderD128 device in /dev/dri/. With that, you can select the "Video Acceleration API" option in Jellyfin's transcode settings page and also select the renderD128 device. I'm not sure if this is important, but I also chmod'd 777 on /dev/dri/renderD128, and installed the vainfo package along the way. Note: you may also need to turn off subtitles in the player, which may have known limitations with Quick Sync and/or FFMPEG.

So far I've tested this with a couple of movies: 
*H264 1080p movie, transcoding from 8mps down to 5mbps. CPU runs at 
*HVEC Main 10 movie, transcoding from 64Mbps down to 5mbps (H264)

Without H/W transcoding, the first movie's software transcoding would bring my Syno CPU to about 98% sustained. The second movie would be unplayable (unless Direct Play'd).

With the above changes, the CPU runs under 25% in both cases. I believe it would be lower if I wasn't also transcoding audio (my hardware doesn't support DTS, so it transcodes DTS audio to MP3).

This is the only testing I've performed, so I can't tell you if it will stand the test of time across all content and configs.

Hope this helps!

**Comment by blinddark** [comment]
please provide a spk for Synology x64 (DS918+)

**Comment by JorgenHedlund** [comment]
More than happy to tryout a Synology build for my DS1813+.

**Comment by idealclover** [comment]
Could test on DS218+.

**Comment by skyzonemc** [comment]
I could test on DS918+

**Comment by tophneal** [comment]
If there's any room for porting to Netgear's ReadyNAS OS, that would be awesome! I'd be happy to help however I can (trying to find information about building apps for their Debian builds, now.)

**Comment by jmooo** [comment]
> Just want to comment for Syno DS918+ users (J3455 CPU with Quick Sync), that I seem to have HW Accelerated transcoding working within the standard Jellyfin docker image (version 10.3.7.0) on the DS918+. I was able to accomplish this by running it in privileged mode (not everyone will be comfortable with this), which exposes the host's /dev directory, including the required renderD128 device in /dev/dri/. With that, you can select the "Video Acceleration API" option in Jellyfin's transcode settings page and also select the renderD128 device. I'm not sure if this is important, but I also chmod'd 777 on /dev/dri/renderD128, and installed the vainfo package along the way.

I was able to accomplish it on my DS918+ using the [linuserver.io image](https://hub.docker.com/r/linuxserver/jellyfin), it _did not_ require running in privileged mode, and _did not_ require the `vainfo` package. I mounted the `/dev/dri/renderD128` device like the docs say (almost, they don't specify the `renderD128` part but you need it), but it _did require_ running `sudo chmod 777 /dev/dri/renderD128` to get HW trancoding actually working. Like you can I see CPU usage dropped from 98% to 25% for a single 1080p stream.

Thanks for tips, hopefully it's a few less steps and a little more secure for other people finding this ticket!

**Comment by rnsc** [comment]
I can test on a DS1618+ if needed :)

**Comment by zippyy** [comment]
i could donate an arm based real synology for testing or a i386/amd64 whitebox xpenology machine running 6.2

**Comment by anthonylavado** [comment]
Hi all,
It's not really "spam", but GitHub doesn't have any good options for this.

We understand you'd like it to come to Synology and other NAS package centers. Hardware isn't the issue, as many people have volunteered, and even offered hardware. To those who have, _thank you_.

The main issue now is getting someone with enough experience and time to help make this a reality. To that end, there isn't anyone currently on the core/packaging teams that knows how to do this. If anyone has any good resources, or is willing to do this, please reach out to us (see [Getting Help](https://jellyfin.org/docs/general/getting-help.html)). We can help provide guidance, but we need help to get started first.

Thanks,
Anthony

**Comment by Uskompuf** [comment]
Has any progress been made here?

**Comment by dkanada** [comment]
nope

**Comment by Foncekar** [comment]
> 
> 
> nope

https://github.com/SynoCommunity/spksrc/pull/3932
You can find one spk here for x64  (jellyfin_x64-6.1_10.5.2-0.spk)

**Comment by koelner21** [comment]
GREAT!!!! Thank you

---
Ich habe diese Nachricht mit der GMX Mail App für Windows erstellt.


-----Ursprüngliche Nachricht-----Von: Foncekar <notifications@github.com>Gesendet: Mittwoch, 1. Juli 2020 11:58An: jellyfin/jellyfin <jellyfin@noreply.github.com>Cc: koelner21 <david.dieter3@gmx.de>; Comment <comment@noreply.github.com>Betreff: Re: [jellyfin/jellyfin] Jellyfin Server Software coming to NAS package centers (like Synology's) (#293)</comment@noreply.github.com></david.dieter3@gmx.de></jellyfin@noreply.github.com>


 

nope

SynoCommunity/spksrc#3932You can find one spk here for x64 (jellyfin_x64-6.1_10.5.2-0.spk)

**Comment by koelner21** [comment]
I think many will want to test it first and because of the ports, you can't just install Jelly because Synology says the port is already in use. Maybe someone can create a manual for it, since you don't want to uninstall Emby on the off-chance that the database is gone.

Thanks.

**Comment by publicarray** [comment]
You install it after you have stopped Emby. You can have both installed but only one running at a time. So you can keep your Database.

**Comment by koelner21** [comment]
No this does Not work. I Must deinstall Emby and then i am able to install jelly, but i cant activate Transcoding and without using Addons it is Bad. This Try was Bad for ne, because Emby Slack does Not work and My database was left. 

Mit freundlichen Grüßen

David Dieter

**Comment by GermanChap** [comment]
> 
> 
> After 10.3.2 lands, I can start looking into this. I don't know that any of the core or support teams have any Synology hardware, so we would need someone to test.
> 
> I have applied here for more details: https://www.synology.com/en-global/support/developer
> 
> I am also looking at https://synocommunity.com/

I know this from like a million years ago, but I am willing to give you access to my synology for testing.
I truly want a Synology Package so if I can make that happen, I will.

**Comment by publicarray** [comment]
You can get a prerelease here: https://github.com/publicarray/spksrc/releases/tag/jellyfin-10.7.0 it requires our own ffmpeg since it's newer than what Synology provides. I need help whith people testing hardware acceleration. Specifically I'm trying to find out if this line is required
https://github.com/SynoCommunity/spksrc/pull/3932/files/ffbad8dfa84b5503269f007c5e37b2956d9a36a8..b9a541502d1e2886129a17947bb9daa1054a9c57#r600104541

I need to test for both DSM6 and 7, unfortunately I haven't figured out how to virtualize/access `/dev/dri/renderD128` on my DSM6 virtual machine yet. DSM7 should work

**Comment by blinddark** [comment]
I can'T install ffmpeg on DSM 7. There is the message, that ffmpeg can't install because it's running as root.

**Comment by publicarray** [comment]
Thanks, sorry I forgot that I compiled ffmpeg myself and that it's not published yet.

**Comment by GermanChap** [comment]
> You can get a prerelease here: https://github.com/publicarray/spksrc/releases/tag/jellyfin-10.7.0 it requires [our own ffmpeg](https://synocommunity.com/package/ffmpeg) since it's newer than what Synology provides. I need help with people testing hardware acceleration. Specifically I'm trying to find out if this line is required
> https://github.com/SynoCommunity/spksrc/pull/3932/files/ffbad8dfa84b5503269f007c5e37b2956d9a36a8..b9a541502d1e2886129a17947bb9daa1054a9c57#r600104541
> 
> I need to test for both DSM6 and 7, unfortunately I haven't figured out how to virtualize/access `/dev/dri/renderD128` on my DSM6 virtual machine yet. DSM7 should work

I would like to start off by saying thank you. Seriously thank you for this.
Secondly, good news. My test movie Wonder Woman 1984 in 4K works without any settings changes.
This is something the Docker Version on the same NAS couldn't do.

Thirdly, unfortunately there is an issue where it doesn't allow you to select directories inside shared folders.

Example: "volume1/video" I cannot go deeper than that.

DSM: 6
NAS: DS3617xs
Test File: 
-Title: Wonder 1984
-Resolution: 3840x2160
-Container: matroska,webm
-Video Codec: hevc
-Audio Codec: eac3
-Video bitrate: 18251.4Kbps

**Comment by publicarray** [comment]
Thanks @GermanChap

> Thirdly, unfortunately there is an issue where it doesn't allow you to select directories inside shared folders.

Did you give sc-jellyfin the correct permissions? Strange I was able to select a sub folder.
How is hardware acceleration?

**Comment by GermanChap** [comment]
> Did you give sc-jellyfin the correct permissions? Strange I was able to select a sub folder.
> How is hardware acceleration? (VAAPI)

Unfortunately installing the package did not create an sc-jellyfin group. Would manually creating it work?

**Comment by B0ngripper** [comment]
> > Did you give sc-jellyfin the correct permissions? Strange I was able to select a sub folder.
> > How is hardware acceleration? (VAAPI)
> 
> Unfortunately installing the package did not create an sc-jellyfin group. Would manually creating it work?
> VAAPI doesn't work sorry.
You can give sc-jellyfin folder permissions in file station. It's a system user, therefore it's not listed under "Users". But you can find it under "Permissions" in folder properties. Unfortunately, i still can't add library folders in the wizard, even if read access is set.

**Comment by publicarray** [comment]
Ok so because how shared folders are implemented you need to give sc-jellyfin (internal-user) or sc-media (group) read permissions for the shared folder from the control panel. You can then use file station to deny any permissions to the sc-jellyfin user for other folders if you wish.

**Comment by publicarray** [comment]
Regarding hardware acceleration you can help me by providing a FFmpeg transcode log file with HW acceleration enabled.

**Comment by GermanChap** [comment]
> 
> 
> Ok so because how shared folders are implemented you need to give sc-jellyfin (internal-user) or sc-media (group) read permissions for the shared folder from the control panel. You can then use file station to deny any permissions to the sc-jellyfin user for other folders if you wish.
> 
> Edit: you may need to restart jellyfin too.

Apply read permissions has fixed the issues of directory depth. Worked like a charm.
Rebooting Jellyfin was required.

**Comment by GermanChap** [comment]
> 
> 
> Regarding hardware acceleration you can help me by providing a FFmpeg transcode log file with HW acceleration enabled.

FFMPEG log for when I try to watch "Wonder Woman 1984" with VAAPI enabled
===============================================================
/videos/30685d61-746b-9586-64f4-c80d1d0e0104/hls1/main/0.ts

{"Protocol":0,"Id":"30685d61746b958664f4c80d1d0e0104","Path":"/volume1/docker/(                                >

/var/packages/ffmpeg/target/bin/ffmpeg -hwaccel vaapi -hwaccel_output_format va                                >


ffmpeg version 4.3.1 Copyright (c) 2000-2020 the FFmpeg developers
  built with gcc 4.9.3 (crosstool-NG 1.20.0) 20150311 (prerelease)
  configuration: --target-os=linux --cross-prefix=/home/spksrc/git-master/spksr                                >
  libavutil      56. 51.100 / 56. 51.100
  libavcodec     58. 91.100 / 58. 91.100
  libavformat    58. 45.100 / 58. 45.100
  libavdevice    58. 10.100 / 58. 10.100
  libavfilter     7. 85.100 /  7. 85.100
  libavresample   4.  0.  0 /  4.  0.  0
  libswscale      5.  7.100 /  5.  7.100
  libswresample   3.  7.100 /  3.  7.100
  libpostproc    55.  7.100 / 55.  7.100
[AVHWDeviceContext @ 0x7daec0] No VA display found for device /dev/dri/renderD1                                >
Device creation failed: -22.
Failed to set value '/dev/dri/renderD128' for option 'vaapi_device': Invalid ar                                >
Error parsing global options: Invalid argument

**Comment by GermanChap** [comment]
So I did some poking around and installed the opkg version of ffmpeg via Easy Bootstrap Installer (found [here](https://www.cphub.net/?id=37) )

Map Jellyfin to this path: /opt/bin/ffmpeg

And VAAPI works. :)

**Comment by GermanChap** [comment]
This path "/dev/dri/renderD128" doesn't exist on my Synology. Not even the "dri" directory exists.
Anyone know how to generate it?

**Comment by publicarray** [comment]
Thanks @GermanChap for your reports! 
Turns out your NAS's CPU doesn't support HW acceleration: https://docs.google.com/spreadsheets/d/1MfYoJkiwSqCXg8cm5-Ac4oOLPRtCkgUxU0jdj3tmMPc (search for DS3617xs)

The file `/dev/dri/renderD128` is created by the kernel driver. So unfortunately this is a hardware limitation.

**Comment by florianbrinkmann** [comment]
Hey @publicarray,

first: thanks for your work! I have a DS220+ and followed the following steps to use your Jellyfin integration with DSM6:

1. Install ffmpeg from SynoCommunity, Version 4.3.1-35.
2. Install your integration from https://github.com/publicarray/spksrc/releases/tag/jellyfin-10.7.0
3. Use File Station to give `sc-jellyfin` read access to my movies folder
4. Enable Hardware Acceleration in the Jellyfin settings (VAAPI)

And it seems to work, I guess, because I do not get any errors when starting a 1080p movie (would I get an error there if Hardware Acceleration would not work?).

**Comment by publicarray** [comment]
Thanks @florianbrinkmann. :+1: Yes you would get an error otherwise (depends on the codec as well obviously since HW/A is only available in a few codecs).

**Comment by florianbrinkmann** [comment]
Okay, cool. It was a codec that is supported by HW/A, so: seems to work fine :)

**Comment by potatoru** [comment]
Thank you for your work! So sad I'm on DSM 7 :(

**Comment by potatoru** [comment]
Thanks for your work! Just tested with DSM 7 on DS420+. No issues noticed yet and HW encoding is working!

**Comment by anthonylavado** [comment]
@publicarray Thanks for your efforts on this. If you need any "official", feel free to send me an email (on GH profile), or drop in to our chat > https://jellyfin.org/contactus

**Comment by gitthangbaby** [comment]
Works on Xpenology DSM6.2.
Couldn't transfer anything from Docker, there are hardcoded paths in library db, and no luck with any db or config file. <PathSubstitute> doesn't work. Always crashes. So don't waste hours, just setup from scratch.

**Comment by JacobMillward** [comment]
Can confirm that it works on my DS920+. Hardware transcoding is working too.

Something to note when using this after using the docker image, is that the leftover data in your browser cache/cookies/etc. seem to cause havoc with the setup process of the new Jellyfin instance if you reuse the port. I encountered an infinite loading screen. Highly reccomend clearing them out as a workaround.

**Comment by redmand** [comment]
Tested on a DS1520+ running DSM 6.2.  Most everything seems to be working for me with the exception of Live TV using a HDHomerun Connect DUO.  The HDHomerun is detected, channel scan works but when I attempt to play any channel the player launches but just spins.  Testing the same with the docker version works perfectly.

**Comment by publicarray** [comment]
@redmand if you would like to share your logs it could tell us what went wrong. 

Thanks @JacobMillward would you be willing to add this information to https://github.com/SynoCommunity/spksrc/wiki/Jellyfin ? Thanks

**Comment by publicarray** [comment]
Thanks @anthonylavado. I've sent you a DM on Twitter.

**Comment by redmand** [comment]
@publicarray logs attached.
[jellyfin_log.txt](https://github.com/jellyfin/jellyfin/files/6318268/jellyfin_log.txt)

**Comment by publicarray** [comment]
@redmand Unfortunately I don't have a DVB, so I don't have a good way to debug this, but I found this interesting snippet in your log `process cannot access the file 'c608cbf3e2604c16a605550d87a0fc00.ts' because it is being used by another process`

```log
[2021-04-15 09:08:58.795 -04:00] [INF] [31] MediaBrowser.MediaEncoding.Encoder.MediaEncoder: "/var/packages/ffmpeg/target/bin/ffprobe" "-analyzeduration 3000000 -i \"http://127.0.0.1:8096/LiveTv/LiveStreamFiles/c608cbf3e2604c16a605550d87a0fc00/stream.ts\" -threads 0 -v warning -print_format json -show_streams -show_format"
[2021-04-15 09:08:59.827 -04:00] [ERR] [41] Jellyfin.Server.Middleware.ExceptionMiddleware: Error processing request: "The process cannot access the file '/volume1/@appstore/jellyfin/var/data/transcodes/c608cbf3e2604c16a605550d87a0fc00.ts' because it is being used by another process". URL "GET" "/LiveTv/LiveStreamFiles/c608cbf3e2604c16a605550d87a0fc00/stream.ts".
```

**Comment by publicarray** [comment]
@redmand This should be fixed with the latest Jellyfin version, building a new version now. Would you be willing to test that one? https://github.com/publicarray/spksrc/releases/tag/jellyfin-10.7.2

**Comment by redmand** [comment]
@publicarray I just tested this latest version.  Problem solved!

**Comment by ylamarche** [comment]
Does hardware decoding work with a graphics card. Example GTX 1050 Ti or Radeon RX 580 ?

**Comment by matt77303** [comment]
Any chance of a armv5 version for DSM 7. I installed a version via SynCommunity and that not longer installs on DSM 7. Moans about port 8095 being used by system. Had to move to Plex and it's no where near as good as jellyfin. Thank you.

**Comment by publicarray** [comment]
@matt77303 can you try this command `sudo servicetool --remove-configure-file --package jellyfin.sc`?

**Comment by matt77303** [comment]
Will do as soon as I get home. Thank you.

**Comment by matt77303** [comment]
Brilliant. That sorted it. Thank you.

Matt

**Comment by ibex-are-goats** [comment]
> I need to test for both DSM6 and 7, unfortunately I haven't figured out how to virtualize/access `/dev/dri/renderD128` on my DSM6 virtual machine yet. DSM7 should work

I was having problems with SynoCommunity Jellyfin and FFMPEG on Synology DSM6 but I found this thread and I'd like to help. I have a DS920+ and you can run your questions by me. The problems I just ran into were: 1) 4K video freezing, phone app stopping, multiple copies of FFMPEG staying alive in background on server, and 2) Uninstalling and reinstalling Jellyfin resulted in the port reserved error, so I can't install it again. https://i.imgur.com/PxlqTPt.png

**Comment by witzker** [comment]
Hi,
on Reddit, I got this link:
https://synocommunity.com/package/jellyfin

But I cannot figure out which one to use for old DS 418 and how to install is without losing data or ruining something

What do you think?

**Comment by DeadMozay** [comment]
> 

https://synocommunity.com/#easy-install
the package will appear in the Package Center

**Comment by witzker** [comment]
Yes 
![image](https://user-images.githubusercontent.com/4120266/180746165-c84c917b-581f-4805-b5ea-9b09aa7b9977.png)
 but it is Version 10.7.7-4
as you see above, I asked for help with Version 10.8.0-5

What have I done wrong?
There is no Update in The Package Center

![image](https://user-images.githubusercontent.com/4120266/180746986-34b76929-c374-4f6e-8486-fef1fa363e7d.png)

**Comment by kaltsoplyn** [comment]
Hi, thank you for your hard work and all. No, really, thank you...

Anyway, DS215j (armada 375) on DSM 6.2.4.25556 Update 6, and Jellyfin doesn't start.
The story is that jellyfin used to work fine, then, after one of my drives crashed, I installed a new one, then I saw that there was an update available (10.8.3-6), I updated and that was that. It just doesn't start.

First of all, is this the right place to post this issue?

If so, here's some more info:
- Latest versions of ffmpeg (4.4.2-46) and .NET 6 (6.0.4-1) installed and running.
- I do have an sc-jellyfin user with read/write permissions everywhere. There is no sc-ffmpeg user, though; I think there used to be one, but I'm not really sure now.
- `synopkg status jellyfin` yields the following message:
`jellyfin package status is stopped but unit is active, it is possible that package encounters an error after it was started successfully`
`Status: [310]`
- `synopkg start jellyfin` yields a package started successfully message, but jellyfin doesn't start (as per above)
- ...  and in `/usr/local/jellyfin/var/` the log shows nothing. just:
`Starting jellyfin command env LD_LIBRARY_PATH=/volume2/@appstore/jellyfin/lib /volume2/@appstore/jellyfin/share/jellyfin --service  --package-name synology  -d /volume2/@appstore/jellyfin/var/data  -C /volume2/@appstore/jellyfin/var/cache  -c /volume2/@appstore/jellyfin/var/config  -l /volume2/@appstore/jellyfin/var/log  -w /volume2/@appstore/jellyfin/web  --ffmpeg /var/packages/ffmpeg/target/bin/ffmpeg`

Any help would be greatly appreciated.

**Comment by oddstr13** [comment]
Looks like this is (partially) solved by the availability on https://synocommunity.com/package/jellyfin
The package there seems up to date.

If you are missing your NAS platform of choice, please search for it on https://features.jellyfin.org/ , upvote if it exists, or create a new feature request if it doesn't.

Requests for
 - [WD NAS](https://features.jellyfin.org/posts/220/port-to-wd-nas-western-digital-pr4100)
 - [FreeNAS](https://features.jellyfin.org/posts/221/freenas-plugin)
